### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_FilesTab_ScreenReader:"Select All/None" Form elements should have an appropriate label associated to it.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3960]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Chrome:Version 68.0.3440.106 
Operating System:Windows10
 
 **Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.    
2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3.Click on Run or right click and select Run to run the notebook.
5.Navigate to the "Folder" icon.
6.Start the Narrator.
**Actual:**
"Select All/None" should have an appropriate label associated to it.
**Expected:**
"Select All/None" should have an appropriate label associated to it.
Element Path
#select-all

 
Snippet
<input type="checkbox" class="pull-left tree-selector" id="select-all">

 
**How to fix
Fix any of the following:**
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
- Form element does not have an implicit (wrapped) <label>
- Form element does not have an explicit <label>
- Element has no title attribute or the title attribute is empty
**UserImpact:**
User will not get wrong information regarding the state of the link. 
**Attachment:**
[A11y_AzureNotebooks_Jupyter_FilesTab_ScreenReader123.pptx](https://github.com/jupyter/notebook/files/2399966/A11y_AzureNotebooks_Jupyter_FilesTab_ScreenReader123.pptx)
![png_attachment](https://user-images.githubusercontent.com/39204909/45800229-d739b100-bccd-11e8-8bce-e14716baaabd.png)

**Comment by takluyver** [comment]
Closed by #4695.
